modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java

```diff
@@ -355,6 +365,27 @@ boolean updateLiveEvent(MediaPackage previousMp, DublinCoreCatalog episodeDC) th
     return true;
   }
 
+  private void createOrUpdatePublicationTracks(Publication publication, Map<String, Track> generatedTracks) {
+    for (String publishedStreamingFormat : publishedStreamingFormats) {
+      Track track = generatedTracks.get(publishedStreamingFormat);
+      if (track != null) {
+        if (publication.getTracks().length > 0) {
+          publication.clearTracks();
+        }
```

Comment:
It's possible to have more than one publishedStreamingFormat, correct? Then, the clearTracks() may delete a track added in the previous iteration of the loop... It would be best to delete the tracks before the loop.

Reply:
Absolutely %-) Stupid me ... thanks again

Reply:
Looks good now! Thanks!